src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -365,13 +384,8 @@ def getFileSystem(
 
         if full_case_name not in self._fs_rotator:
             if not os.path.exists(full_case_name):
-                if self._fs_rotator.atCapacity():
-                    self._fs_rotator.dropOldestFileSystem()
-
-                EnkfFs.createFileSystem(full_case_name)
-
-            new_fs = EnkfFs(full_case_name, read_only)
-            self._fs_rotator.addFileSystem(new_fs, full_case_name)
```

Comment:
This is very strange behavior, why does, essentially a `setter` create a whole new filesystem?